### PR TITLE
Fix hyperlink on Pod Topology Spread Constraints page

### PR DESCRIPTION
**Pull request by kadtendulkar**
Follow up on #35643

Updated page [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints)

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ utilization.
 
 You can set [cluster-level constraints](#cluster-level-default-constraints) as a default,
 or configure topology spread constraints for individual workloads.
-
+
```
Comment (Sea-n) [inline content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:18]: Is this space intended?

**Comment by mk46** [comment]
> Follow up on #35643
> 
Please update this as `Fixes #35643`. that will autoclose the linked issue when your PR merged. See step 7 in https://kubernetes.io/docs/contribute/new-content/open-a-pr/#open-a-pr

**Comment by kadtendulkar** [comment]
Hello @divya-mohan0209 and @mehabhalodiya please review PR.

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 2]
```diff
@@ -442,7 +442,7 @@ topology spread constraints are applied to a Pod if, and only if:
 
 Default constraints can be set as part of the `PodTopologySpread` plugin
 arguments in a [scheduling profile](/docs/reference/scheduling/config/#profiles).
-The constraints are specified with the same [API above](#api), except that
+The constraints are specified with the same [API above](#example-topologyspreadconstraints-with-nodeaffinity), except that
```
Comment (annajung) [inline content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:445]: Would it make sense to point to the #topologyspreadconstraints-field instead?

Comment (kadtendulkar) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:445]: Yes @annajung we can replace it with #topologyspreadconstraints-field , should I proceed with changes?

**Comment by kadtendulkar** [comment]
Hello @annajung  and @mehabhalodiya  since I was facing issues after later commits, I have raised new PR #36396

**Comment by Sea-n** [comment]
Close as it replaced by another PR.

- #36396

/close

**Pull request closed without merging**